packages/lib/src/vis/shared/PanZoomMesh.tsx

```diff
@@ -23,154 +13,13 @@ interface Props {
 }
 
 function PanZoomMesh(props: Props) {
```

Comment:
For ease of refactoring and testing in the stories, I kept `PanZoomMesh` and its `Props` for now. 

Now, it only calls the new meshes `PanMesh` and `ZoomMesh`.